app/components/operator-form/template.hbs

```diff
@@ -1,6 +1,20 @@
 <div class="container input-form">
 	<div class="row">
 		<div class="col-sm-12 col-xs-12 input-section">
+			<div class="input-form-label">Include this operator?</div>
+				<div class="input-form-field">
+					{{#if isIncluded}}
```

Comment:
Any need for having two different sets of buttons? Might work to just have one instance of each button, withe the conditional `active` class.

Also, this is where you could bind to `operator.include_in_changeset`, if you wanted to just use that, instead of an extra `isIncluded` property on the component.